c_emulator/riscv_platform.c

```diff
@@ -48,6 +48,12 @@ mach_bits plat_rom_base(unit u)
 mach_bits plat_rom_size(unit u)
 { return rv_rom_size; }
 
+mach_bits plat_zicbom_cache_block_size()
```

Comment:
Why are there two separate configurable block sizes? I assume this will always be the cache block size so if both extensions are supported the values must be the same?
`plat_cache_block_size()`. Also this function signature is missing the `unit u` argument.